adafruit_hcsr04.py

```diff
@@ -54,39 +61,51 @@ class HCSR04:
 
     ::
 
-        with HCSR04(trig, echo) as sonar:
+        import time
+        import board
+
+        import adafruit_hcsr04
+
+        sonar = adafruit_hcsr04.HCSR04(trigger_pin=board.D2, echo_pin=board.D3)
+
+
+        while True:
             try:
-                while True:
-                    print(sonar.dist_cm())
-                    sleep(2)
-            except KeyboardInterrupt:
+                print((sonar.distance,))
+            except RuntimeError:
+                print("Retrying!")
                 pass
+            time.sleep(0.1)
     """
-    def __init__(self, trig_pin, echo_pin, timeout_sec=.1):
+    def __init__(self, trigger_pin, echo_pin, *, timeout=0.1):
         """
-        :param trig_pin: The pin on the microcontroller that's connected to the
+        :param trigger_pin: The pin on the microcontroller that's connected to the
             ``Trig`` pin on the HC-SR04.
         :type trig_pin: str or microcontroller.Pin
         :param echo_pin: The pin on the microcontroller that's connected to the
             ``Echo`` pin on the HC-SR04.
         :type echo_pin: str or microcontroller.Pin
-        :param float timeout_sec: Max seconds to wait for a response from the
+        :param float timeout: Max seconds to wait for a response from the
             sensor before assuming it isn't going to answer. Should *not* be
             set to less than 0.05 seconds!
         """
-        if isinstance(trig_pin, str):
-            trig_pin = getattr(board, trig_pin)
+        if isinstance(trigger_pin, str):
+            trigger_pin = getattr(board, trigger_pin)
         if isinstance(echo_pin, str):
             echo_pin = getattr(board, echo_pin)
```

Comment:
How about removing this. It adds a second way to do something that will be confusing.

Reply:
totally OK - will do!